tests/v2/validation/workloads/workload_test.go

```diff
@@ -272,7 +279,7 @@ func (w *WorkloadTestSuite) TestWorkloadUpgrade() {
 	require.NoError(w.T(), err)
 	require.NotEmpty(w.T(), logRollback)
 
-	validateDeploymentUpgrade(w, namespace.Name, upgradeDeployment, "6", "ubuntu", 2)
+	validateDeploymentUpgrade(w, namespace.Name, upgradeDeployment, "6", "ubuntu", 4)
```

Comment:
is this expected? asking as I dont see any other change in the test to the container count?

Reply:
This is me trying to figure out a timing issue with this particular test and the expected containers not becoming ready. I'm going to revert this change because Leo is working on a way to make the test less flakey.